src/main/java/today/sesac/shoutify/post/Controller/PostCommandController.java

```diff
@@ -0,0 +1,62 @@
+package today.sesac.shoutify.post.Controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import today.sesac.shoutify.global.response.ApiResponse;
+import today.sesac.shoutify.post.dto.request.PostCreateRequest;
+import today.sesac.shoutify.post.dto.response.PostCreateResponse;
+import today.sesac.shoutify.post.service.PostCommandService;
+
+@RestController
+@RequestMapping("/api/v1/posts")
+public class PostCommandController {
+    private final PostCommandService postCommandService;
+
+    public PostCommandController(PostCommandService postCommandService) {
+        this.postCommandService = postCommandService;
+    }
+
+    /**
+     * 게시물 작성
+     */
+    @PostMapping()
+    public ResponseEntity<ApiResponse<PostCreateResponse>> savePost(
```

Comment:
P2
여기 뿐만 아니라 전체적으로 ResponseEntity로 따로 감싼 이유가 있는지 궁금합니다. 
ApiResponse<PostCreateResponse> -> 이것만 반환하도록 해도 똑같지 않나요? 확인해 보시고 공유 부탁드립니다~
```suggestion
    public ApiResponse<PostCreateResponse> savePost(
```

Reply:
뭔가 2번이나 감싸는게 이상하다 했는데 HttpEntity를 상속받아서 상태코드를 지정해줄 수 있네요. 저희는 공통응답을 사용하니 딱히 필요는 없어보입니다! 그리고 ResponseEntity를 사용할 경우 빌더 패턴을 사용하는걸 권장한다고 하네요!

Reply:
오 근데 이거 builder 패턴을 어떻게 쓰는지 아시나여?

Reply:
생성자의 인자가 긴 경우에 chain형태를 사용해서 대신 쓸 수 있는 패턴...으로 알고 있는데 더 알아보겠습니다헤헤